pkg/chart/chart.go

```diff
@@ -24,14 +23,14 @@ type Metadata struct {
 func GenerateChartConfig(basePath string, chartName string) ([]byte, error) {
 	cmd := exec.Command("schemadocs", "generate", "values.schema.json", "-o", "README.md", "-l", "linear")
 	cmd.Dir = basePath + HELM_CHARTS_FOLDER + chartName
-	output, err := cmd.CombinedOutput()
+	_, err := cmd.CombinedOutput()
 	if err != nil {
-		return nil, microerror.Maskf(CouldNotGenerateChartFileError, err.Error(), string(output))
+		return nil, err
```

Comment:
Maybe you want to keep the logs? you could use something like `fmt.Errorf` instead or just returning the error?

Reply:
I would be in favour of building a habit of wrapping errors here, maybe something along the lines of 

```golang
if err != nil {
    return fmt.Errorf("failed to generate chart %q %w", string(output), err)
}
```

This costs little but means we can unwrap the error trace if / should we need it?